src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -300,7 +298,7 @@ protected void onPostExecute(Void result) {
       protected void executeAction(@Nullable Void parameter) {
         for (long chatId : selectedConversations) {
           if (chatId == DcChat.DC_CHAT_ID_DEADDROP) {
-            dcContext.marknoticedContact(getListAdapter().getDeaddropContactId());
+            dcContext.marknoticedChat(DcChat.DC_CHAT_ID_DEADDROP);
```

Comment:
ftr, this is not implemented in core, not sure, however, if that is needed at all.

i think, the idea was to get the contact-request "out of sight", however, implicitly marking as noticed when one taps "archive" or "delete" (case below) might also be a bit weird. not sure.

Reply:
I deprecated `marknoticed_contact()` in core and removed it from the C bindings as it's not needed for deciding on contact requests anymore; so, we should probably implement this functionality in `marknoticedChat()`

Reply:
yip, either adapt or remove this functionality. i commented a bit more detailed at https://github.com/deltachat/deltachat-android/pull/1781 - let's continue the discussion there.